app/controllers/surveys_controller.rb

```diff
@@ -13,6 +13,7 @@ def send_survey
       return render json: {error: :survey_not_found}, status: :not_found
     end
 
+    @survey.generate_models
```

Comment:
This is going to be much easier once we add the published flag so I'm going to add a TODO for now and come back to it.